defaults/defaults_darwin.go

```diff
@@ -0,0 +1,40 @@
+// +build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package defaults
+
+const (
+	// DefaultRootDir is the default location used by containerd to store
+	// persistent data
+	DefaultRootDir = "/var/lib/containerd"
```

Comment:
Wondering; would it make sense to;

- use `/var/run/xxxxx`  for Linux as well? I know `/run` is the "new" location, but any (all) Linux platforms will still have a symlink (`/var/run/` -> `/run`)
- keep the exported consts shared, but have a non-exported const for unix, darwin?

train of thought for the last one is that it's means we only need to maintain a single GoDoc, and make sure that the "API" (exported) consts stay in sync (see https://github.com/opencontainers/selinux/pull/104)